specs/catalog-schema-registry.spec/src/main/scripts/io/aklivity/zilla/specs/catalog/schema/registry/schema/schema.registry.schema.patch.json

```diff
@@ -115,7 +117,31 @@
                     "additionalProperties": false
                 }
             },
-            "additionalProperties": false
+            "additionalProperties": false,
+            "if":
+            {
+                "properties":
+                {
+                    "url":
+                    {
+                        "pattern": "^https:\/\/"
+                    }
+                }
+            },
+            "then":
+            {
+                "properties":
+                {
+                    "tls": true
+                }
+            },
```

Comment:
Does this still validate the contents of `tis` properly when `https` url is present?